.github/workflows/integration_tests.yml

```diff
@@ -5,14 +5,13 @@ on:
     branches:
       - main
   pull_request:
-    branches:
-      - main
   workflow_dispatch:
 
 jobs:
 
   swift-button-functional-test:
-    runs-on: macOS-latest
+    runs-on: macOS-12
+    if: "!github.event.pull_request.head.repo.fork"
```

Comment:
Is it worth adding a comment about why this is needed? The code isn't necessarily self-documenting and future self may not know or remember what issue is being addressed here.

Reply:
For sure.  Done.